Sage/carbon
# feat: add new VerticalMenu component - FE-5619

### Proposed behaviour

This PR introduces a new VerticalMenu component

Closes Sage/design-system#12

### Checklist




#### QA


### Testing instructions

Check FE-5619 ticket for acceptance criteria + answered questions that I had for the UX team.

Create test stories and add Cypress tests.
